python_modules/dagster/dagster/_core/storage/event_log/sql_event_log.py

```diff
@@ -590,6 +647,28 @@ def enable_secondary_index(self, name):
                     .values(migration_completed=datetime.now())
                 )
 
+    def _apply_tags_table_joins(
```

Comment:
There's only a single join going on here, right (with multiple join conditions)? If so, `_apply_tags_table_joins` would be more accurate.

Reply:
Hmm, did you mean `_apply_tags_table_join`?

Reply:
Oops yes